python/rascaline/tests/utils/correlate_density.py

```diff
@@ -160,19 +160,7 @@ def get_norm(tensor: TensorMap):
 )
 @pytest.mark.parametrize(
     "frames, nu_target, angular_cutoff, selected_keys",
-    [
-        (
-            h2_isolated(),
-            3,
-            None,
-            Labels(
-                names=["spherical_harmonics_l", "inversion_sigma"],
-                values=np.array([[0, 1], [4, 1], [5, 1]]),
-            ),
-        ),
-        (h2o_isolated(), 2, 5, None),
-        (h2o_periodic(), 2, 5, None),
-    ],
+    [(h2o_periodic(), 2, 3, None)],
 )
 def test_so3_equivariance(frames, nu_target, angular_cutoff, selected_keys):
```

Comment:
Can you remove all arguments & parametrize, and instead create `frame = h2o_periodic()` & friends inside the function?

Reply:
Also, do we still have a test checking `selected_keys` elsewhere?

Reply:
Yes - `test_correlate_density_angular_selection`